tox.ini

```diff
@@ -1,12 +1,19 @@
 [tox]
-envlist = py27, py35, py36, py37, flake8
+envlist = py38, py39
```

Comment:
is dropping py27 a requirement to get this to work? We have kept it specifically for MacOS users where python2.7 is still the default install.

Reply:
ack, I can restore that.  I had originally dropped it to sync with the other projects python ver support, but I didn't take into consideration that someone would need 2.7.  Update imminent.